ui/raidboss/data/04-sb/raid/o3n.js

```diff
@@ -253,27 +258,31 @@ export default {
         standOnFrogTile: {
           en: 'Stand on frog tile',
           de: 'Auf Frosch-Fläche stehen',
+          fr: 'Tenez-vous sur la grenouille',
```

Comment:
"Tenez-vous" c'est moche, ça fait un peu vieillot.
"Placez-vous" ou "Allez sur", c'est mieux, non ?

Pareil pour les autres du coup.

Reply:
moi aussi. je suis d'accord?
"allez sur" est mieux?

Reply:
C'est également celui que je préfère.

Reply:
Slt, disons que c'est la traduction la plus fidèle, pour moi. C'est aussi celle que nous avons mis pour toutes les autres, qui commencent par "`stand in`". Si tu y tiens vraiment, "`placez-vous`" me parais un bon compromis. Il faudra que je modifie les autres traductions aussi. Pour la cohérence générale.

Reply:
Rien n'empêche d'avoir des différences sur certains fights, pas besoin de tout modifier. Et là, il faut aller sur la grenouille.

Reply:
J'ai remplacé par `Placez-vous`. Je verrais pour les quelques autres linké plus haut. Il n'y en a pas beaucoup, je le ferais plus tard sur une autre requête.